commit 76dda21a7dc612d7755c287348c52574ac35738f (refs/changes/55/2455/2)
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2017-08-24T14:19:46-07:00 (2 years, 1 month ago)
    
    DOCKER-1091 Cannot login into Artifactory private docker registry
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/package.json b/package.json
index c220bb5..96b024c 100644
--- a/package.json
+++ b/package.json
@@ -11,7 +11,7 @@
     "bunyan": "1.8.8",
     "cmdln": "3.2.1",
     "dashdash": "1.10.0",
-    "docker-registry-client": "3.2.6",
+    "docker-registry-client": "3.2.8",
     "effluent-logger": "git+https://github.com/joshwilsdon/effluent-logger.git#d662f161a07f94045ad2afb45442931511c40e51",
     "expiring-lru-cache": "2.1.0",
     "extsprintf": "1.2.0",
